data/json/items/melee.json

```diff
@@ -779,18 +779,21 @@
         "to_hit": 2
     },
     {
-        "type":"GENERIC",
+        "type":"AMMO",
         "id": "concrete",
         "symbol": "%",
         "color": "light_gray",
-        "name": "bag of concrete",
-        "name_plural": "bags of concrete",
-        "description": "A bag of concrete, ready to be used in a construction project.",
+        "name": "concrete",
+        "name_plural": "concrete",
+        "description": "Some concrete, ready to be used in a construction project.",
+        "container" : "bag_canvas",
         "price": 10000,
         "material": "stone",
-        "weight": 1000,
+        "weight": 585,
```

Comment:
Actually I've just removed weight of a bag from that, to keep weight consistent for spawned bags of concrete. How much should a concrete or mortar itself actually weigh?